build/cocos2d_libs.xcodeproj/project.pbxproj

```diff
@@ -17490,21 +17490,21 @@
 				HEADER_SEARCH_PATHS = "";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.1;
 				LIBRARY_SEARCH_PATHS = (
-					"\"$(SRCROOT)/../external/png/prebuilt/ios\"",
```

Comment:
I made a PR to include tvOS: https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin/pull/286